news/_posts/2020-04-27-Polymake.jl-0.4-has-been-tagged.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Polymake.jl 0.4 has been tagged
+author: Marek Kaluba
+---
+
+`Polymake.jl` v0.4.0 has been tagged!
```

Comment:
"tagged" is technical jargon. Not everybody reading our website will be familiar with that. It also makes it unclear whether the code has also been released.

I would suggest to use here (and in the `title:` field above) something that's easier to understand for those people; e.g. "has ben released" or "is available now" or "is ready" or so.

Reply:
Also, in the `title` it says `0.4.0` and here `v0.4.0`. Pick one?

Reply:
I'd also suggest to use a hyperlink to Polymake.jl so that people can quickly find the repo. Alternatively, it could be to your documentation, if that is available online?